.github/workflows/ci.yml

```
@@ -49,4 +49,35 @@ jobs:
                     :tada: Splice release `${{ env.GSF_DEVNET_VERSION }}` is ready for deployment on DevNet, and is available from the GSF
                     :ship: ⅔ of Super Validator nodes have upgraded to this release, including the GSF Super Validator node
                     *SV node status here*: https://sync.global/sv-network
-                    *Release notes here*: https://docs.dev.sync.global/release_notes.html
\ No newline at end of file
+                    *Release notes here*: https://docs.dev.sync.global/release_notes.html
+
+      - name: Announce to validator-announce@lists.sync.global
+        env: 
+          VALIODATOR_OPERATORS_GROUP_ID: "133138"
+        run: |
+          LOGIN_RESPONSE_COOKIES=$(curl -XPOST -c cookies.txt "https://lists.sync.global/api/v1/login" \
+            -d "email=${{ secrets.GROUPS_IO_USER }}&password=${{ secrets.GROUPS_IO_PASS }}")
+          CSRF_TOKEN=$(echo $LOGIN_RESPONSE_COOKIES | jq -r '.user.csrf_token')
+          DRAFT_ID=$(curl "https://lists.sync.global/api/v1/newdraft" \
+            -b "cookies.txt" \
+            -d "draft_type=draft_type_post" \
+            -d "group_id=${VALIODATOR_OPERATORS_GROUP_ID}&csrf=${CSRF_TOKEN}" \
+            | jq -r '.id')
+
+          # Replace version in template
+          sed "s/{{VERSION}}/${{ env.GSF_DEVNET_VERSION }}/g" ./ci/message_template.html > message.html
```

[nit] `envsubst` does exactly the same but look a bit cleaner (`GSF_DEVNET_VERSION` will have to be specified under `env:` above)
```suggestion
          cat ci/message_template.html | envsubst > message.html
```